### ISSUE TITLE
Fix second-run bug

### ISSUE THREAD

**Issue by migurski** [openaddresses/machine#53]
Recently started seeing this exception, after the first post-S3 run. The bug does not look to be in the Lexington source, but rather a result of that source finishing first.

```
Traceback (most recent call last):
  File "/usr/local/bin/openaddr-process", line 9, in <module>
    load_entry_point('OpenAddresses-Machine==1.1.1', 'console_scripts', 'openaddr-process')()
  File "/usr/local/lib/python2.7/dist-packages/openaddr/process_all.py", line 47, in main
    process(s3, paths.sources, run_name)
  File "/usr/local/lib/python2.7/dist-packages/openaddr/process_all.py", line 113, in process
    results = jobs.run_all_process_ones(source_files, 'out', source_extras)
  File "/usr/local/lib/python2.7/dist-packages/openaddr/jobs.py", line 114, in run_all_process_ones
    completed_path, result = result_iter.next(timeout=report_interval)
  File "/usr/lib/python2.7/multiprocessing/pool.py", line 659, in next
    raise value
IOError: [Errno 2] No such file or directory: '/20150119/us-sc-lexington.json'
```

**Comment by migurski** [comment]
Making some progress on this one. We haven’t really been delivering on some of the OA documentation regarding caches, fingerprints, and so on.

**Comment by migurski** [comment]
Fixed in 7ee7468 and 7d6c4d8.
